db_stress_tool/db_stress_test_base.cc

```diff
@@ -1839,6 +1840,21 @@ void StressTest::Open() {
       options_.new_table_reader_for_compaction_inputs = true;
     }
   }
+  if (FLAGS_sst_file_manager_bytes_per_sec > 0 ||
```

Comment:
Just to clarify, if user does not specify these two newly-added flags, `options_` will not have an sst file manager created here, but `SanitizeOptions` will create an SstFileManager anyway with default values for these two knobs. Should we say this loud while defining these two flags?

Reply:
Yeah good point, I didn't notice it in `SanitizeOptions()` as I forgot the code organization lately. Will fix everything that implies `SstFileManager` is only created by setting these options.